README.md

```diff
@@ -1,3 +1,34 @@
-HTMl and CSS3 Module 1 project:  Positioning and Floating Elements.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Author: Mariano Abilleira.
+# Project Name
+
+HTML and CSS3 Module 1 project: Positioning and Floating Elements.
+
+The main objective of this project is to create a clone of a New York Times article page. The concepts that we look to apply on it are:
+
+_ Flexbox/Grid.<br/>
+_ Box Model.<br/>
+_ Positioning and embedding of images.
```

Comment:
[OPTIONAL] You can use `- `  or `* `  instead of underscore "_" in order to make it a list type ✏️ 
(Remember, it needs to space after `- `  or `* ` )